README.md

```diff
@@ -4,16 +4,107 @@
+The `value` is not mandatory if you are using a reference.
+
+If you want to delete a property you can invoke the contextual menu on the specific property you want to delete and simply select `Delete Property`. The same procedure applies when you want to edit a property just that you select `Edit Property` instead.
+
+### Environment Variables
+
+An environment variable can contain required information for the Data Virtialization. There are no predefined variables added when creating a new VDB. If you want to create an environment variable you can do that by invoking the contextual menu on the `Environment` entry inside the `Data Virtualization` view as shown in the screenshot below.
+
+![Create new Environment Variable Context Menu](./images/dv_create_envvar.png)
+
+Similar to the datasource properties you can choose one of the 3 available options of `New Variable` contextual menu items.
+
+`New Variable` - This will create a new key / value pair stored directly inside the YAML file.
+
+`New Variable from ConfigMap` - This will create a reference to an entry inside a ConfigMap.
+
+`New Variable from Secret` - This will create a reference to an entry inside a Secret.
+
+When using a ConfigMap or Secret you will be asked for a `reference name` which is the name of the ConfigMap or Secret and the `reference key` which is the key under which the value is stored in the ConfigMap / Secret.
+The `value` is not mandatory if you are using a reference.
+
+If you want to delete a variable you can invoke the contextual menu on the specific variable you want to delete and simply select `Delete Variable`. The same procedure applies when you want to edit a variable just that you select `Edit Variable` instead.
+
+### Modifying the Schema
+
+When creating a new VDB file you are supplied with a schema template which creates a simple view to be used in the data virtualization runtime. If you want to make modifications to that schema you can use the `Edit Schema` contextual menu on the `Schema` node of your VDB inside the `Data Virtualization` view.
+
+![Edit Schema Context Menu](./images/dv_edit_schema.png)
+
+This option will open the schema (the DDL section of the YAML) in an editor for you. 
+
+(__WORK IN PROGRESS__) In future that editor will provide code completion, syntax coloring and more intellisense to help you create the schema faster. 
+
+### The extension log output channel
+
+The Data Virtualization extension provides its own logging output channel to help users understanding problems that might occur during usage. This log might be useful for reporting issues back. (see the screenshot below)
+
+![Log Output Channel](./images/dv_output_channel.png)
+
+
+## How to contribute
+
+Contributions to the project are welcome. You can either report issues, ask for new features, provide pull requests for code or documentation files.
```

Comment:
would be nice to link the JIRA project

Reply:
shouldn't that be part of the Contributing.md then?

Reply:
agree. it might be better to place it in Contributing.md and only point to the contributing.md file

Reply:
This might be another consistency thing across all extension readmes